cvat/apps/engine/static/engine/js/shapes.js

```diff
@@ -1745,6 +1746,9 @@ class RaysController extends PolyShapeController {
     }
 
     rotate(points, rotationPoint, targetPoint) {
+        if (!this._model._rotateRays)
```

Comment:
roateRays is a constant, so the function is a constantly the identity. Can't it be removed? 
Or at least the call to this be removed if we plan to use this function elsewhere.

moveVanishingPoint is also the identity now

Reply:
> Most of my comments are stylistic, and not essential if this blocks annotators. The only real one I would like addressed is keeping the vanishing point: left and right lines need to intersect somewhere, and if it is marked it is better. The only change is that the center line does not need to intersect exactly in the same point.

What is benefit to keep the vanishing point visualisation for left and right lines? We don't have any requirements on it and don't provide its value in an output.